### DISCUSSION TITLE
Github Actions Not Triggering or Updating Status?

### DISCUSSION THREAD

**Discussion by michealroberts** [vueuse/vueuse#1482] [General]
I'm just raising the awareness to the core team on an issue I am experiencing with the automated test runners and PRs.

I've recently pushed some changes to the following PR, essentially I have rebased main into the branch and I've resolved all remaining conflicts. 

However, the build actions pipeline is not being triggered in the Actions panel.

The PR in question is here: https://github.com/vueuse/vueuse/pull/1332

And the recently rebasing and merged resolutions is in this commit: https://github.com/vueuse/vueuse/pull/1332/commits/8b691b894ac8e580cfa792f50efb1df62ae682fd

Yet, the actions are not being triggered and the status of the PR is still "_This branch has conflicts that must be resolved_"

**Comment by michealroberts** [comment]
FOA @antfu

**Comment by antfu** [comment]
It seems to be a bug of GitHub, we experienced that in multiple repos at that time, try push a new commit to re-trigger it
